model/client.go

```diff
@@ -1289,7 +1289,7 @@ func (c *Client) MigrateClusterInstallation(request *MigrateClusterInstallationR
 		return nil
 
 	default:
-		return errors.Errorf("failed with status code %d", resp.StatusCode)
+		return errors.Errorf("Failed with status code %d", resp.StatusCode)
```

Comment:
I might have not been clear enough in my previous comment. The errors should stay lowercase, only the logged messages start with upper. Sorry for confusion.